plugins/inputs/mqtt_consumer/README.md

```diff
@@ -67,6 +81,13 @@ to use them.
   ## Connection timeout for initial connection in seconds
   # connection_timeout = "30s"
 
+  ## Interval and ping timeout for keep-alive messages
+  ## The sum of those options defines when a connection loss is detected.
+  ## Note: The keep-alive interval needs to be in second granularity e.g. 1m
```

Comment:
This can only be integer-multiples of seconds so everything that is below a second will produce an error and all fractions of a second will be dropped... Appreciate a better formulation! :-)

Reply:
Sounds like there are two criteria, how about: The interval must be larger than 1 second and fractions of a seconds are dropped.